web-registry/public/_redirects

```diff
@@ -2,4 +2,6 @@
 
 /projects  /projects/wilmot  200
 /projects/impactag  /projects/wilmot  200
-/projects/impactag/admin /projects/wilmot/admin 200
\ No newline at end of file
+/projects/impactag/admin /projects/wilmot/admin 200
+
+https://registry.regen.network/*  https://app.regen.network/:splat 301!
```

Comment:
instead of prior change, we'll try out this "domain-level redirect" using the _redirects file syntax. presumably the redirects we already had in here were working, so i'm assuming doing the domain-level redirect in this file as well, will work.

https://docs.netlify.com/routing/redirects/redirect-options/#domain-level-redirects